lib/mixins/lambdas.js

```diff
@@ -5,6 +5,12 @@ module.exports = function lambdas(req, res) {
     renderField() {
       return function renderFieldMixin() {
         const mixin = this.mixin;
+        if (this.useWhen) {
+          if (req.sessionModel.get(this.useWhen.field) !== this.useWhen.value) {
```

Comment:
Not sure I follow you? If useWhen is implicitly true, then the below check is executed (which will fail) and the value is unset, but it should only be unset if useWhen is defined, and condition not met